src/test/ui/const-generics/invalid-const-arg-for-type-param.stderr

```diff
@@ -13,11 +18,14 @@ LL | struct S;
 LL |     S.f::<0>();
    |       ^ method not found in `S`
 
-error[E0107]: wrong number of const arguments: expected 0, found 1
-  --> $DIR/invalid-const-arg-for-type-param.rs:8:9
+error[E0107]: wrong number of const arguments for `S`: expected 0, found 1
+  --> $DIR/invalid-const-arg-for-type-param.rs:8:5
    |
+LL | struct S;
+   | --------- defined here, with 0 const parameters
+...
 LL |     S::<0>;
-   |         ^ unexpected const argument
+   |     ^   - help: consider removing this const argument
```

Comment:
This will also require the span to cover `::<` and `>` for the removal to be correct.

Reply:
improved 🎉